Src/Common/CorrelationIdLookupHelper.cs

```diff
@@ -136,18 +136,6 @@ public bool TryGetXComponentCorrelationId(string instrumentationKey, out string
                             {
                                 this.GenerateCorrelationIdAndAddToDictionary(instrumentationKey, appId.Result);
                             }
-                            catch (AggregateException ex)
```

Comment:
yes - that's just refactoring / cleanup. That's why that's in a separate commit.